### ISSUE TITLE
Internal and API results being limited to 100 

### ISSUE THREAD

**Issue by pilonull** [theotherp/nzbhydra2#571]
When I do an internal or API search (through sonarr or radarr) NZBHydra is applying a limit to these searches that are being sent to the indexers' API, for example:

`NzBNooB: Calling https://www.nzbnoob.com/api?apikey=APIKEYt=search&extended=1&password=1&cat=2000&**limit=100**&offset=0 `

So it's only returning 100 results for any given search. This is mainly a problem for me with internal searches. When I search something, only 100 results are loaded from any particular indexer and the "load more" button is grayed out. 

Another snippet from the log:
```
NzBNooB: Indexer's response indicates a total of 230750 results but actually only 100  were returned
NzBNooB: Successfully executed search call in 1816ms with 100 total results
```

I cannot for the life of me figure out why the "limit=100" is being applied. 
I tried starting fresh with a new NZBHydra install, I thought that fixed it and I go to do an internal search and it's only returning 100 results and the "load more" button is grayed out once again. 

I looked in the config file and found 
`  loadLimitInternal: 100`

I tried messing around with this value, but it appears to reset itself every time NZBHydra is restarted. 

I'm sure this is probably some setting that I'm overlooking but I've spent like 4 hours on this now and I can't figure it out. 

From Config>searching setting section:
I've tried "Ignore load Limiting internally" both on and off, but I do not have load limiting enabled on any of my indexers.
I've tried "Display all retrieved results" Both on and off and that does not appear to fix my problem. 
I've tried Display 1000 results per page instead of just 100 and it still only loads 100. 

I've restarted NZBHydra between all of these setting changes as well. 

I've always been on the latest version, it updates automatically, but the version I am using is 2.22.2
I do not know if this started with the update on the 20th or if this is a result of me messing around with settings last night. But like I said, I tried to start completely fresh and am running in to this problem again. 

Basically, from what I can tell, NZBHydra is limiting the searching API calls to 100 results by itself in when it searches out to my indexers and I do not know how to fix it. 

Any help is greatly appreciated. If need be I can attach my entire log file but thought it might be somewhat easy to narrow down based on the information I have given.

**Comment by theotherp** [comment]
I'm on mobile so just a short answer for now: indexers limit is the amount
of results returned per query to 100. Even if you use a higher value you
still only get 100.

**Comment by pilonull** [comment]
In the past, as in, a couple days ago, it was working normally. I would search something in NZBHydra, it would return a list of 100 results, and the "load more" button in the top would load another 100. Now the load more button is grayed out and it only returns 100 results. I'm unable to load more than 100 results at a time. I hope I'm, explaining it understandably. 

Thanks for your reply

**Comment by theotherp** [comment]
Sorry, misunderstood the main issue. I'll take a look next week.

**Comment by pilonull** [comment]
Cool man I appreciate it, it's probably some dumb thing I did but I don't know how to fix it. I even tried starting fresh and it's still doing it so I'm at a loss.
Hope you have a good weekend, thanks again!

**Comment by fhorv** [comment]
just upgraded from an older version and noticing the same

**Comment by pilonull** [comment]
It's fixed! Thank you so much!
